pkgs/development/interpreters/tcl/mk-tcl-derivation.nix

```diff
@@ -48,7 +49,8 @@ let
     # Run tests after install, at which point we've done all TCLLIBPATH setup
     doCheck = false;
     doInstallCheck = attrs.doCheck or ((attrs ? doInstallCheck) && attrs.doInstallCheck);
```

Comment:
Maybe more similar to the new lines for installCheckInputs and nativeInstallCheckInputs:

```nix
   doInstallCheck = attrs.doCheck or (attrs.doInstallCheck or false);
```

Or maybe instead of (or in addition to) `false`, use the `config.doCheckByDefault` ?

Reply:
done